test/fixtures/iframe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h2>Iframe</h2>
+    <script>
+      window.request = () => {
```

Comment:
I wasn't successful in writing a reliable Puppeteer await logic if the request happens on load. Waiting for elements is extremely flaky.